packages/peregrine/lib/hooks/useApolloContext.js

```diff
@@ -1,10 +1,14 @@
 import { useContext } from 'react';
-import { ApolloContext } from 'react-apollo/ApolloContext';
+import { ApolloContext } from 'react-apollo';
 
 /**
  * A function that provides access to the [Apollo client]{@link https://www.apollographql.com/docs/react/api/apollo-client} API.
  *
  * @return {ApolloClient} The Apollo client for this application
  * @kind function
  */
-export const useApolloContext = () => useContext(ApolloContext);
+export const useApolloContext = () => {
+    const { client } = useContext(ApolloContext) || {};
```

Comment:
Apollo's context value seems to have changed in order to accommodate some private query data (maybe cached results?). We could pass the new raw context object, but we'd have to change the consumers, but I think that'd be a breaking change, and they should only need the client anyway.

If a consumer *really* needs the other private members, it can consume `ApolloContext` directly with `useContext`.

Reply:
I agree with your reasoning, but I think we should include that as a note in the comment block.